circle/benches/cfft.rs

```diff
@@ -0,0 +1,81 @@
+use std::any::type_name;
+
+use criterion::measurement::Measurement;
+use criterion::{criterion_group, criterion_main, BenchmarkGroup, BenchmarkId, Criterion};
+use p3_baby_bear::BabyBear;
+use p3_circle::{CircleDomain, CircleEvaluations};
+use p3_dft::{Radix2Bowers, Radix2Dit, Radix2DitParallel, TwoAdicSubgroupDft};
+use p3_field::TwoAdicField;
+use p3_matrix::dense::RowMajorMatrix;
+use p3_mersenne_31::Mersenne31;
+use rand::distributions::{Distribution, Standard};
+use rand::thread_rng;
+
+fn pretty_name<T>() -> String {
+    let name = type_name::<T>();
+    let mut result = String::new();
+    for qual in name.split_inclusive(&['<', '>', ',']) {
+        result.push_str(qual.split("::").last().unwrap());
+    }
+    result
+}
+
+fn bench_lde(c: &mut Criterion) {
+    let log_n = 18;
+    let log_w = 8;
+
+    let mut g = c.benchmark_group("lde");
+    g.sample_size(10);
+    lde_cfft(&mut g, log_n, log_w);
+    lde_twoadic::<BabyBear, Radix2Dit<_>, _>(&mut g, log_n, log_w);
+    lde_twoadic::<BabyBear, Radix2DitParallel, _>(&mut g, log_n, log_w);
+    lde_twoadic::<BabyBear, Radix2Bowers, _>(&mut g, log_n, log_w);
```

Comment:
Probably should keep these here long term since they seem more fitting in the `dft` crate (which has some similar benches), though if it's convenient we can keep them while you're working on this